diagnostics/ENSO_MSE/MSE/get_clima_in.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import os.path

```

Comment:
get_clima_in and get_data_in can be combined by defining arguments for input file paths